### ISSUE TITLE
Navigation landmark should not be labelled as "navigation"

### ISSUE THREAD

**Issue by afercia** [WordPress/twentytwentyone#470]
**Describe the bug**

Screen readers already announce `<nav>` elements as "navigation".  Labelling a nav element with a string that contains the word "navigation" is redundant:

`<nav aria-label="Footer Navigation" class="footer-navigation">`

The above is announced as:

"Footer Navigation, navigation".

The word "Navigation" should be removed. That would leave us with only the word "Footer", which would be confusign for a navigation element (there's already a "footer" landmark).

I'd suggest to just rename this nav to "Secondary menu" or, if this menu is meant for a specific purpose e.g. social links or whatever else, use a more specific name. With the first option, the primary menu should be renamed to "Primary menu".
